### ISSUE TITLE
Is the project dead?

### ISSUE THREAD

**Issue by SmuSmu** [flourishlib/flourish-classes#227]
I wonder if you still working on flourish. The roadmap says a stable release will show up end of 2012. :-()

**Comment by netcarver** [comment]
@SmuSmu Things are pretty much dead here - but look at imarc's fork ([here](https://github.com/imarc/flourish-classes)) for an alternative version that Matthew has been working on for a while.

**Comment by wbond** [comment]
Yes, the project (as started by me) is effectively dead. iMarc maintained it some after I left the company, but they since have forked it, although I don’t believe there are any plans for futher development.

This is mostly because:

 1. I don’t work with PHP anymore
 2. It has been 9 years since the project started, and the PHP landscape has changed significantly since then

There are certainly bugs that exist in Flourish, however I still use it when I need to implement a solution in PHP that is small and secure. There are still many features (that I care about) that don’t seem to have equivalents in moderns frameworks, however I imagine those things aren’t important to the majority of PHP developers.

I fixed some issues with PHP 7 recently (for my own use), but decided not to publish them as open source. This is because I don’t have the free time, nor motivation to provide support for the project anymore.  If anyone is keen on those, I’d be happy to push up a branch that you can pull from, but I won’t be engaging bug reports, etc.

**Comment by SmuSmu** [comment]
@wbond Sad to hear that. liked it a lot.

@netcarver thanks for the link. But I think it`s dead too there.

**Comment by khamer** [comment]
@SmuSmu We occasionally add bug fixes to our branch as we need them, but aren't doing any new development to [imarc/flourish-classes](https://github.com/imarc/flourish-classes) or adding any functionality.

We still have many projects built and running on Flourish, but we're moving away from Flourish for new work.

**Comment by uBizzy** [comment]
@SmuSmu - great question! I also use parts of flourishlib on several projects I do and also had to move forward on on several "classes" that started to be some how buggy.

@wbond - First of all a big THANK YOU! Your approach creating flourishlib is one of the best around and it definitely define now some of my developments. So if you push a branch with your fixes, I would be happy to look at them, use them and if I have the time add some on my end to.

**Comment by wbond** [comment]
@uBizzy Here you are: https://github.com/flourishlib/flourish-classes/tree/php7-compat-plus-misc-fixes

**Comment by uBizzy** [comment]
Thanks! Will for sure take a deep look on that. Also fell free to contact me if you need.

A great 2017 for you and your families!

**Comment by BrendonKoz** [comment]
Just hopping in here with thanks and praise. I've personally found Flourish's SMTP handling to, honestly, be about the only decent solution out there aside from, possibly, Zend - but Zend's requirements make popping classes in to another project rather annoying. I know I'm not the only one (@netcarver can attest) since email plugins on the ProcessWire CMS use that class too. Great work, @wbond, and thank you.
